<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewerWindow</class>
 <widget class="QMainWindow" name="ViewerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string> Ventana Cliente</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="push_Start">
      <property name="maximumSize">
       <size>
        <width>125</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
      <property name="icon">
       <iconset theme="play">
        <normaloff/>
       </iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="push_Stop">
      <property name="maximumSize">
       <size>
        <width>125</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="Push_Pausa">
      <property name="maximumSize">
       <size>
        <width>125</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Pausa</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="Salir">
      <property name="maximumSize">
       <size>
        <width>125</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Salir</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QLabel" name="label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>50000</width>
        <height>10000</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>536</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>&amp;Archivo</string>
    </property>
    <widget class="QMenu" name="menuAbrir">
     <property name="title">
      <string>Abrir</string>
     </property>
     <addaction name="actionAbrirImagen"/>
     <addaction name="separator"/>
     <addaction name="actionAbrirVideo"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuAbrir"/>
    <addaction name="separator"/>
    <addaction name="actionCapturar"/>
    <addaction name="separator"/>
    <addaction name="actionComenzar_a_enviar"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>A&amp;yuda</string>
    </property>
    <addaction name="actionAcercaDe"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_Preferencias">
    <property name="title">
     <string>&amp;Editar</string>
    </property>
    <addaction name="actionPreferencias"/>
    <addaction name="separator"/>
    <addaction name="actionAjustes_Conexion"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menu_Preferencias"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAbrirImagen">
   <property name="text">
    <string>AbrirImagen</string>
   </property>
  </action>
  <action name="actionAbrirVideo">
   <property name="text">
    <string>AbrirVideo</string>
   </property>
  </action>
  <action name="actionAcercaDe">
   <property name="text">
    <string>AcercaDe</string>
   </property>
  </action>
  <action name="actionCapturar">
   <property name="text">
    <string>Capturar</string>
   </property>
  </action>
  <action name="actionPreferencias">
   <property name="text">
    <string>Preferencias</string>
   </property>
  </action>
  <action name="actionAjustes_Conexion">
   <property name="text">
    <string>Ajustes Conexi√≥n</string>
   </property>
  </action>
  <action name="actionComenzar_a_enviar">
   <property name="text">
    <string>Comenzar a enviar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
